src/Teams/Controllers/ManageTasksController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Teams.Services;
+
+namespace Teams.Controllers
+{
+    public class ManageTasksController : Controller
+    {
+        private readonly IManageTasksService _manageTasksService;
+        private readonly IAccessCheckService _accessCheckService;
+
+        public ManageTasksController(IManageTasksService manageTasksService, IAccessCheckService accessCheckService)
+        {
+            _manageTasksService = manageTasksService;
+            _accessCheckService = accessCheckService;
+        }
+
+        [HttpGet]
+        [Authorize]
+        public async Task<IActionResult> GetTaskByIdAsync(int teamId, int taskId)
+        {
+            if (await _accessCheckService.OwnerOrMemberAsync(teamId))
```

Comment:
please invert if statement

Reply:
and also else won't be needed
if have no access will be nesting to return error, otherwise correct flow without nesting